README.md

```diff
@@ -1,2 +1,122 @@
-# defenz-alerts
-This script is used to send scheduled Defenz alerts by using Defenz APIs
+# Defenz Timely Alerts
+This script is used to generate and send block reports for a specific period of time. 
+This script can be scheduled by a scheduler e.g. cron to be run at specific times.
+
+`NOTE`: To run this script you need Python 3.6+
+
+## How to install
+Clone this repository:
+```shell script
+git clone https://github.com/InternetNZ/defenz-alerts.git
+```
+Install the dependencies:
+```shell script
+cd defenz-alerts
+pip3 install -r requirements.txt
+```
+
+## Set the configs
+Before running the script for the first time you need to configure the script. The config file
+is located in ``DEFENZ_ALERTS_DIR/defenz_alerts/config.ini``. If the config file does not exist 
+creat one from the sample one:
+
+```shell script
+cd DEFENZ_ALERTS_DIR/defenz_alerts/
+cp config.sample.ini config.ini
+```
+
+The config file should look like below:
+
+```ini
+[DEFENZ]
+USERNAME =
+PASSWORD =
+CLIENT_ID =
+CLIENT_SECRET =
+REPORT_EMAIL =
```

Comment:
`Report Email is required and hasn't been provided. Neither by arguments nor by config file!` Testing locally this is forcing me to include a fall back email... perhaps make this optional.

Reply:
We don't want send email if it's not provided?

Reply:
Initially, it makes sense to me that `REPORT_EMAIL` is a required parameter because the report is the outcome of the detection and we do not have another way of alerting. So, if an email is provided what is the reason to run the script?

As we cannot guarantee that a report email exists for the network, the catch-all email is needed.

Reply:
As @camfindlay suggested it might be good for testing/debugging purpose.